frontend/.prettierrc.json

```
@@ -0,0 +1,8 @@
+{
+    "trailingComma": "all",
+    "tabWidth": 4,
+    "semi": false,
+    "singleQuote": true,
+    "arrowParens": "always"
+}
```

```suggestion
{
    "tabWidth": 4,
    "semi": false,
    "arrowParens": "avoid"
}
```

I suggest we use the `tabWidth` and `semi` as this and avoid the parenthesis around singe argument functions. And we can keep the rest as default. 

[docs](http://json.schemastore.org/prettierrc)